src/Test/Integration/Model/Service/ImportOrderTest.php

```diff
@@ -20,33 +20,43 @@
  * @license   http://www.apache.org/licenses/LICENSE-2.0 Apache License, Version 2.0
  */
 
+declare(strict_types=1);
+
 namespace Shopgate\Import\Test\Integration\Model\Service;
 
+use PHPUnit\Framework\TestCase;
 use Shopgate\Base\Tests\Bootstrap;
 use Shopgate\Base\Tests\Integration\SgDataManager;
+use ShopgateLibraryException;
+use Shopgate\Import\Test\Integration\Data\Payment\Braintree\CreditCard;
 
 /**
  * @coversDefaultClass Shopgate\Import\Model\Service\Import
+ *
+ * @magentoAppIsolation enabled
+ * @magentoDbIsolation  enabled
+ * @magentoAppArea      frontend
  */
-class ImportOrderTest extends \PHPUnit\Framework\TestCase
+class ImportOrderTest extends TestCase
 {
     /** @var \Shopgate\Import\Helper\Order */
-    protected $orderClass;
+    private $orderClass;
     /** @var \Shopgate\Import\Model\Service\Import */
-    protected $importClass;
+    private $importClass;
     /** @var array - list of created orders to clean up */
-    protected $orderHolder = [];
+    private $orderHolder = [];
     /** @var \Magento\Sales\Api\OrderRepositoryInterface */
-    protected $orderRepository;
+    private $orderRepository;
     /** @var SgDataManager */
-    protected $dataManager;
+    private $dataManager;
 
-    public function setUp()
+    public function setUp(): void
     {
         $objectManager         = Bootstrap::getObjectManager();
```

Comment:
interestingly we use a different bootstrap here, does it work with the default mage one? I think the idea will be to move away from our own bootstrap.

Reply:
Yep, works.